src/Pages/Login/Login.js

```diff
@@ -2,83 +2,99 @@ import React, { Component } from "react";
+      method: "POST",
+      body: JSON.stringify({
+        email: this.state.email,
+        password: this.state.pw,
+      }),
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        if (res.token) {
+          localStorage.setItem("aesopToken", res.token);
+          this.props.history.push("/");
+          console.log(res.token);
+        }
+      });
+  };
 
-    // 회원가입용
-    handleSignUp = (e) => {
-        this.props.history.push('/Signup');
-    }
+  // 회원가입용
+  handleSignUp = (e) => {
+    this.props.history.push("/signup");
+  };
 
   render() {
     return (
-        <div className="Login">
-            <div className="modalBody">
-            <div className="introduceBox">
-                <h2 className="loginTitle">안녕하세요.</h2>
-                <div className="loginSub">유효한 이메일 주소를 입력하세요</div>
-            </div>
-            <div className="formText">
-                <label>
-                    <input onChange={this.hadleValueID} className="formTextInput" name="email" placeholder="이메일 주소"  />
-                    {/* <span className="FormTextLabel">이메일 주소</span> 이부분은 애니메이션이 요구됨. 해결방안이 시급. */}
-                </label>
-            </div>
-            <div className="formText">
-                <input onChange={this.hadleValuePW} className="formTextInput" type="Password" placeholder="비밀번호"/>
-            </div>
-            <button onClick={this.hadleBtn} type="button" className={this.state.email.length > 5 && this.state.pw.length > 5 ? "activeBTN" : "unactiveBTN"}>
-                <div className="btnContent">
-                    로그인
-                </div>
-            </button>
-            <button onClick={this.handleSignUp} type="button" className="signUpcolor">
-                <div className="signUpBtn">
-                    회원가입
-                </div>
-            </button>
-        </div>
+      <div className="Login">
+        <div className="modalBody">
+          <div className="introduceBox">
+            <h2 className="loginTitle">안녕하세요.</h2>
+            <div className="loginSub">유효한 이메일 주소를 입력하세요</div>
+          </div>
+          <div className="formText">
+            <label>
+              <input
+                onChange={this.hadleValueID}
+                className="formTextInput"
+                name="email"
+                placeholder="이메일 주소"
+              />
+              {/* <span className="FormTextLabel">이메일 주소</span> 이부분은 애니메이션이 요구됨. 해결방안이 시급. */}
+            </label>
+          </div>
+          <div className="formText">
+            <input
+              onChange={this.hadleValuePW}
+              className="formTextInput"
+              type="Password"
+              placeholder="비밀번호"
+            />
+          </div>
+          <button
+            onClick={this.hadleBtn}
+            type="button"
+            className={
+              this.state.email.length > 5 && this.state.pw.length > 5
+                ? "activeBTN"
+                : "unactiveBTN"
```

Comment:
```suggestion
              email.length > 5 && pw.length > 5
                ? "active btn"
                : "btn"
```
1. 구조분해 할당 해주세요
2. 대문자 이름은 상수 변수에만 사용해주세요
3. active를 기준으로 active라는 상태가 변하는 것이기 때문에 btn과 active btn으로 토글하는게 좋습니다. 띄어쓰기로 분리하는 이유는 두 버튼이 동일한 속성을 공유할 경우 코드를 재사용할 수 있기 때문입니다